jscomp/ounit_tests/ounit_cmd_tests.ml

```diff
@@ -276,14 +276,14 @@ let rec y = A y;;
       OUnit.assert_bool __LOC__
         (Ext_string.contain_substring should_err.stderr "contravariant")
     end;
-    __LOC__ >:: begin fun _ ->
```

Comment:
Would you delete instead of commenting out?

Reply:
https://github.com/rescript-lang/rescript-compiler/pull/6199/commits/8d8bf81a75c5378a863149cc2d81390a0b9ef336